### ISSUE TITLE
Callback function is not called when closing fast

### ISSUE THREAD

**Issue by TCastelein** [limonte/sweetalert2#96]
When closing a swal very fast the callback function attached is not called.

Take for example the swal button in your own example:
```js
swal({
  title: 'Are you sure?',
  text: "You won't be able to revert this!",
  type: 'warning',
  showCancelButton: true,
  confirmButtonColor: '#3085d6',
  cancelButtonColor: '#d33',
  confirmButtonText: 'Yes, delete it!',
  closeOnConfirm: false
}).then(function(isConfirm) {
  if (isConfirm) {
    swal(
      'Deleted!',
      'Your file has been deleted.',
      'success'
    );
  }
})
```

If you open this swal and very rapid press the Enter key the next swal is not loaded. You can test this by making a button and selecting this then press enter rapidly 2 times. First wil activate the swall second will press confirm.

It seems like you have to wait for the animation to finish before the callback function is activated.

So the bug is callback function is not always called when closing swal.

**Comment by limonte** [comment]
Thank you @TCastelein for submitting this issue!

I think it's the same bug as in #93 

Update swal2 to latest and try again.

**Comment by TCastelein** [comment]
Thank you for your reply my apologies for not reading the other issue as closely. I thought the promise was another issue entirely. 

The update seems to fix the issue i had with not fireing the callback when closed too quickly

Issue closed, duplicate of #93
